### ISSUE TITLE
Move accessibility into own top-level category

### ISSUE THREAD

**Issue by paulirish** [GoogleChrome/lighthouse#1875]
Rob this probably makes sense to do in your mega-PR adding all the new checks.

**Comment by patrickhulce** [comment]
FWIW I don't think this really needs to be done prior to the report refactor. I've already done it in my config v2 branch with little effort.

**Comment by paulirish** [comment]
Add patrick to this so he can include the category/topic thing as part of config v2.

**Comment by ebidel** [comment]
Can we consider this taken care of then? v1 supports it now after Rob's PR and Patrick has it in v2.

**Comment by paulirish** [comment]
yup. good call.
